src/game/player_super_sonic.c

```diff
@@ -1,4 +1,867 @@
+#include "global.h"
 #include "core.h"
+#include "trig.h"
+#include "sakit/globals.h"
+#include "sakit/camera.h"
+#include "sakit/player.h"
```

Comment:
I think we can consider importing `camera` and `player` from the actual game implementation? From the rom ordering, it seem the global is declared in the game part of the rom, but still referenced from the sakit portion

Reply:
I wouldn't yet do that, but it might make sense in the future, once we have more data to verify that.